encord/client.py

```diff
@@ -589,6 +598,9 @@ def list_label_rows(
             "edited_after": edited_after,
             "label_statuses": label_statuses,
             "shadow_data_state": shadow_data_state.value if shadow_data_state else None,
+            "include_uninitialised_labels": include_uninitialised_labels,
```

Comment:
Taking a step back here (and bringing the private chat to public) - why do we want to give the option to users to retrieve a label row without a label_hash? Can they use it in some way?

Ideally, I'd like to keep things simple and just pretend those do not exist as far as the client is aware